modules/images/webp-uploads/fallback.js

```diff
@@ -0,0 +1,67 @@
+( function( d ) {
+	window.webpUploadsFallbackWebpImages = function( media ) {
+		for ( var i = 0; i < media.length; i++ ) {
+			try {
+				var ext = media[i].media_details.sources['image/jpeg'].file.match( /\.\w+$/i );
+				if ( ! ext || ! ext[0] ) {
+					continue;
+				}
+
+				var images = d.querySelectorAll( 'img.wp-image-' + media[i].id );
+				for ( var j = 0; j < images.length; j++ ) {
+					images[j].src = images[j].src.replace( /\.webp$/i, ext[0] );
+					var srcset = images[j].getAttribute( 'srcset' );
+					if ( srcset ) {
+						images[j].setAttribute( 'srcset', srcset.replace( /\.webp(\s)/ig, ext[0] + '$1' ) );
+					}
+				}
+			} catch ( e ) {
+			}
+		}
+	};
+
+	var restApi = d.getElementById( 'webpUploadsFallbackWebpImages' ).getAttribute( 'data-rest-api' );
```

Comment:
Just curios on the reason why passing `document` as a variable and not `window` as well.

Reply:
`window` is used only once in the `fallback` script and passing it via arguments won't help to make the script code smaller. Instead, it will make the script code bigger because it will add additional variable declaration (in the function params) and variable call (to define a new function).

Reply:
Thanks for your answer.

I don't see any difference tho by not passing document and just using it directly as an argument.